docs/CLI/5_developer_guide.md

```diff
@@ -1,5 +1,18 @@
 # Developer Guide
 
+## Prerequisites
+
+- Go 1.18+
+- Git
+
```

Comment:
Nit: I would generally write out “repository” over “repo” so it errs on the side of more professional writing.

Reply:
Agree.